src/components/Questions.tsx

```diff
@@ -19,7 +19,10 @@ interface QuizProps {
   chooseAnswer: boolean;
   points: number;
   choicesArr: string[][];
-  checkAnswer: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  selectedOption: string;
+  selectOption: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  checkAnswer: () => void;
+  answerButtonsRef: React.MutableRefObject<any>;
```

Comment:
it looks like this will resolve the eslint error

```suggestion
  answerButtonsRef: React.MutableRefObject<HTMLUListElement | null>;
```